src/centos/scripts/start-marklogic.sh

```diff
@@ -172,17 +234,23 @@ elif [[ "${MARKLOGIC_INIT}" == "true" ]]; then
     fi
 
     log "Initialzing MarkLogic on ${HOSTNAME}."
-
-    curl -s --anyauth -i -X POST \
-        -H "Content-type:application/json" \
+    TIMESTAMP=$(curl --anyauth -m 20 -s --retry 5 \
```

Comment:
Better to have timeout set to same value either 20 or 30. Not both is different places

Reply:
Agreed, I missed that when the change to 30s was made. Fixed now.